### PR TITLE
Speedup Driver Compilation

### PR DESCRIPTION
**Pull request by stimpsonsg**
Changes to farm out USE statements to individual subpackages, which should speed up the overall build time, and definitely improves Driver build time.

### CODE REVIEW

File: src/Futility.f90 [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  USE MemProf
+  USE IO_Strings
+  USE IOutil
+  USE BLAS
+  USE FileType_Fortran
+  USE FileTYpe_Input
+  USE FileType_Log
+  USE ExpTables
+  IMPLICIT NONE
+!
```
Comment (aarograh) [inline src/Futility.f90]: Not sure we need this ! here.

Comment (bscollin) [reply src/Futility.f90]: So I'm a little confused on why you picked these... other than they are in driver.  Just doesn't seem very intuitive.

Comment (aarograh) [reply src/Futility.f90]: Actually, ben's right... I saw modules like BLAS that are already lumped together and thought you covered everything... you should probably list all the fortran modules

Comment (stimpsonsg) [reply src/Futility.f90]: Okay...I just included the ones Driver used...yep

**Pull request closed without merging**